platform/src/main/resources/config/anchor-config-default-values.yaml

```diff
@@ -0,0 +1,487 @@
+  ## transaction).
+  #
+  auth_timeout: 900
+
+  ## @param: jwt_timeout
+  ## @type: integer
+  ## Set the timeout in seconds of the authenticated JSON Web Token. An expired JWT will be rejected.
+  ## This is the timeout period after the client has authenticated.
+  #
+  jwt_timeout: 86400
+
+  ## @param: omnibus_account_list
+  ## A comma-separated list of omnibus accounts (comma separated).
+  ## The SEP-10 omnibus account is described at:
+  ##     https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0010.md#memos
+  #
+  omnibus_account_list:
+
+  ## @param: require_known_omnibus_account
+  ## @type: bool
+  ## Whether to require authenticating clients to be in the omnibusAccountList
+  #
+  require_known_omnibus_account: false
+
+sep12:
+
+  ## @param: enabled
+  ## @type: bool
+  ## Whether to enable SEP-12 for this instance of AP server.
+  #
+  enabled: false
+
+sep24:
+
+  ## @param: enabled
+  ## @type: bool
+  ## Whether to enable SEP-24 for this instance of AP server.
+  #
+  enabled: false
+
+  ## @param: interactive_jwt_expiration
+  ## @type: integer
+  ## Set the timeout in seconds of the JSON Web Token returned with the embedded interactive url of the SEP-24 process.
+  ## An expired JWT will be rejected.
+  ##
+  ## If the interactive flow needs to access the platform server, the interactive process must finish within
+  ## the specified timeout period.
+  #
+  interactive_jwt_expiration: 3600
+
+  ## @param: interactive_url
+  ## @type: string
+  ## The interactive URL where the platform server will redirect to start the SEP-24 interactive flow.
+  #
+  interactive_url: http://localhost:8080/sep24/interactive
+
+sep31:
+
+  ## @param: enabled
+  ## @type: bool
+  ## Whether to enable SEP-31 for this instance of AP server.
+  #
+  enabled: false
+
+  ## @param: payment_type
+  ## @default: STRICT_SEND
+  ## Used to determine how amount_in is calculated from amount in the POST /transaction call
+  ## @supported_values:
+  ##     STRICT_SEND: amount_in = amount
+  ##     STRICT_RECEIVE: amount_in = amount + fee
+  payment_type: STRICT_SEND
+
+  ## @param: deposit_info_generator_type
+  ## @default: self
+  ## Used to choose how the SEP-31 deposit information will be generated, which includes the
+  ##     deposit address, memo and memo type.
+  ## @supported_values:
+  ##     self: the memo and memo type are generated in the local code, and the distribution account is used for the deposit address.
+  ##     circle: the memo and memo type are generated through Circle API, as well as the deposit address.
+  ##     api: the memo and memo type are generated through calling the anchor's GET /unique_address endpoint.
+  #
+  deposit_info_generator_type: self
+
+sep38:
+  ## @param: enabled
+  ## @type: bool
+  ## Whether to enable SEP-38 for this instance of AP server.
+  #
+  enabled: true
+
+
+##########################
+## Metrics Configuration
+##########################
+metrics:
```

Comment:
Can we table the documentation improvements to later PRs?

Reply:
Can you add a ticket for us to do another round of review on the ConfigMap and update its documentation?

Some comments in this review that relate to documenting the config map file:
* https://github.com/stellar/java-stellar-anchor-sdk/pull/516#discussion_r974744238
* https://github.com/stellar/java-stellar-anchor-sdk/pull/516#discussion_r974745691 (this one)
* https://github.com/stellar/java-stellar-anchor-sdk/pull/516#discussion_r974746059
* https://github.com/stellar/java-stellar-anchor-sdk/pull/516#discussion_r974742496

Reply:
Thanks Marcelo.

Reply:
Can you elaborate a little bit more about which part of ConfigMap requires review in order for me to create the ticket? 

It will be ok if @marcelosalloum can create the ticket too. 😺

Reply:
Created in https://github.com/stellar/java-stellar-anchor-sdk/issues/586.